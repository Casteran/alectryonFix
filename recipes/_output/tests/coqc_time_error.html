<?xml version="1.0" encoding="utf-8" ?>
<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" class="alectryon-standalone" xml:lang="en" lang="en">
<head>
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<meta name="generator" content="Docutils 0.16: http://docutils.sourceforge.net/" />
<title>Errors from coqc -time</title>
<link rel="stylesheet" href="alectryon.css" type="text/css" />
<link rel="stylesheet" href="docutils_basic.css" type="text/css" />
<link rel="stylesheet" href="pygments.css" type="text/css" />
<script type="text/javascript" src="alectryon.js"></script>
<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/IBM-type/0.5.4/css/ibm-type.min.css" integrity="sha512-sky5cf9Ts6FY1kstGOBHSybfKqdHR41M0Ldb0BjNiv3ifltoQIsg0zIaQ+wwdwgQ0w9vKFW7Js50lxH9vqNSSw==" crossorigin="anonymous" />
<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/firacode/5.2.0/fira_code.min.css" integrity="sha512-MbysAYimH1hH2xYzkkMHB6MqxBqfP0megxsCLknbYqHVwXTCg9IqHbk+ZP/vnhO8UEW6PaXAkKe2vQ+SWACxxA==" crossorigin="anonymous" />
<meta name="viewport" content="width=device-width, initial-scale=1">
</head>
<body>
<div class="alectryon-root alectryon-centered"><div class="alectryon-banner">Built with <a href="https://github.com/cpitclaudel/alectryon/">Alectryon</a>, running Coq+coqc-time. Bubbles (<span class="alectryon-bubble"></span>) indicate interactive fragments: hover for details, tap to reveal contents. Use <kbd>Ctrl+‚Üë</kbd> <kbd>Ctrl+‚Üì</kbd> to navigate, <kbd>Ctrl+üñ±Ô∏è</kbd> to focus. On Mac, use <kbd>‚åò</kbd> instead of <kbd>Ctrl</kbd>.</div><div class="document" id="errors-from-coqc-time">
<h1 class="title">Errors from coqc -time</h1>

<p>An incomplete file is an error with <tt class="docutils literal">coqc <span class="pre">-time</span></tt>.
To compile:</p>
<pre class="literal-block">
alectryon --coq-driver=coqc_time coqc_time_error.rst &gt; coqc_time_error.out 2&gt;&amp;1; \
  echo &quot;exit: $?&quot; &gt;&gt; coqc_time_error.out
    # ReST ‚Üí HTML; produces ‚Äòcoqc_time_error.out‚Äô
</pre>
<pre class="alectryon-io highlight"><!-- Generator: Alectryon --><span class="alectryon-wsp"><span class="kn">Goal</span> <span class="kt">True</span>.</span></pre><div class="system-messages section">
<h1>Docutils System Messages</h1>
<div class="system-message">
<p class="system-message-title">System Message: ERROR/3 (<tt class="docutils">tests/coqc_time_error.rst</tt>, line 14)</p>
Driver Coq+coqc-time (coqc) exited with code 1:
   Error: There are pending proofs: Unnamed_thm.
</div>
</div>
</div>
</div></body>
</html>
