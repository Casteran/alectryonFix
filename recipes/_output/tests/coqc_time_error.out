tests/coqc_time_error.rst:14: (ERROR/3) Driver Coq+coqc-time (coqc) exited with code 1:
   Error: There are pending proofs: Unnamed_thm.

exit: 13
